app/realtime.php

```diff
@@ -32,41 +41,81 @@
 ini_set('default_socket_timeout', -1);
 Swoole\Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
 
-$server = new Server("0.0.0.0", 80);
+$server = new Server('0.0.0.0', 80);
+
+$server->set([
+    'websocket_compression' => true,
+    'package_max_length' => 81920
+]);
+
 $subscriptions = [];
 $connections = [];
 
-$server->on("workerStart", function ($server, $workerId) use (&$subscriptions, &$connections) {
-    Console::success('Worker '.++$workerId.' started succefully');
+$register = new Registry();
```

Comment:
$register should be available from init.php, unless your'e resting it on purpose?

Reply:
No, will use the one from `init.php` 👍🏻